app/routes/events/components/EventAdministrate/Attendees.js

```diff
@@ -107,7 +107,10 @@ export default class Attendees extends Component<Props, State> {
     if (error) {
       return <div>{error.message}</div>;
     }
-    const showUnregister = moment().isBefore(event.startTime);
+    const showUnregister =
+      moment().isBefore(event.startTime) ||
+      moment().isBefore(event.unregistrationCloseTime) ||
+      moment().isBefore(event.registrationCloseTime);
```

Comment:
What's the reasoning behind this? Shouldn't admins be able to admin unregister after registration closes?

Anyways, this doesn't really make sense. If `unregistrationCloseTime` is before `registrationCloseTime`, the button will show, but won't work.

Reply:
What is your preferred way then? Doesn't really matter to me.

Reply:
This logic is dead (together with the GDPR logic here) anyway I guess, so it doesn't really matter

Reply:
> Shouldn't admins be able to admin unregister after registration closes?

They can with this equally much as before

Reply:
Backend don't care tho., so it doesn't matter